compose/mpp/demo/src/uikitMain/kotlin/bugs/IosBugs.kt

```diff
@@ -25,5 +25,6 @@ val IosBugs = Screen.Selection(
     KeyboardPasswordType,
     UIKitRenderSync,
     DispatchersMainDelayCheck,
-    StartRecompositionCheck
+    StartRecompositionCheck,
+    SelectionHandlePadding,
```

Comment:
Is it iOS only?

Reply:
Please move it to common, there is no specific uikit API usage here.

Reply:
Code sample not iOS only. So, moved to commonMain sourceSet in TextFields section

Reply:
Done